spec/export/index.md

```diff
@@ -78,6 +84,24 @@ Export data from a FHIR server whether or not it is associated with a patient. T
 
   Specifies the format of the optional OperationOutcome response to the kick-off request. Currently, only application/fhir+json is supported.
 
+  **Example OperationOutcome**
```

Comment:
Why is this here? It looks like an error that might be returned in response to fetching an exported file, but it's sitting in the "kickoff" portion of the spec.

Reply:
@ricky1uphealth This issues still needs to be addressed

Reply:
Gotcha. I moved it to the correct section. It's fixed here https://github.com/HL7/bulk-data/pull/7/files#diff-985cef1e2f9cd85436f27813aa6de763R206